components/checkboxes/spec/Checkboxes.stories.mdx

```diff
@@ -24,7 +25,7 @@ Let users select one or more options by using the checkboxes component.
       label={<h1 className="govuk-heading-l">Which types of waste do you transport?</h1>}
       name="waste"
       options={[
-        { value: 'carcasses', label: 'Waste from animal carcasses' },
+        { value: 'carcasses', label: <Fragment><b>Waste</b> from animal carcasses</Fragment> },
```

Comment:
We generally want the examples to match the ones from GDS. So we should make this change.